backends/qualcomm/qnn_preprocess.py

```diff
@@ -104,7 +104,8 @@ def preprocess(
             else:
                 raise RuntimeError(f"{node.op} is not supported in Qnn")
         qnn_context_binary = qnn_manager.Compile(
-            [py_op_wrapper.GetOpWrapper() for py_op_wrapper in py_op_wrapper_list]
+            qnn_manager.GetGraphNames()[0],
```

Comment:
what does this line mean?

Reply:
Currently we will have multiple graphs inside one `QnnManager`, the exposed APIs need graph name as an identifier to manipulate. But in this stage, there will be only one graph for processing where the graph name comes from compiler specs.